### ISSUE TITLE
Why lr does not change when using AdamW?

### ISSUE THREAD

**Issue by Jacobew** [facebookresearch/detr#158]
I've seen from the training log that the learning rate of the network remains unchanged in each step, except for a drop at args.lr_drop. 
Does the AdamW optimizer dynamically change the lr in the training process?

**Comment by alcinos** [comment]
Hi @Jacobew 
We use a step schedule, where the lr is divided by 10 after args.lr_drop epochs, as you noticed.
The rest of the times the lr is constant, in particular, we are not using any kind of warmup, as sometimes done in NLP.
AdamW doesn't "dynamically change the lr".

I believe I have answered your question, and as such I'm closing this. Feel free to reach out if you have further concerns.
